index.html

```diff
@@ -312,5 +204,30 @@ <h2>Contact me</h2>
         </form>
       </section>
     </main>
+    <div class="detail-container">
+      <div class="detail-popup hide">
+        <img
+          class="close-button"
+          src="assets/detailCloseIcon.png"
+          alt="Close button"
+        />
+        <div class="title"></div>
+
+        <ul class="info"></ul>
+        <p class="desc"></p>
+        <div class="right">
+          <ul class="skills"></ul>
+          <hr class="hr" />
+          <button>
+            <span>See live</span>
+            <img src="assets/seeLiveIcon.png" alt="See live" />
+          </button>
+          <button>
+            <span>See source</span>
+            <img width="16px" src="assets/Vector.png" alt="See source" />
+          </button>
+        </div>
+      </div>
+    </div>
```

Comment:
Nice job so far👍 But just as [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) for this milestone, we are to generate the popup content and project section dynamically with javascript using the array of objects we have created to store data about our projects😉 Kindly take away this hardcoded popup and generate it dynamically in your `js` file👍
<img width="824" alt="generate popup and projects section dynamically" src="https://user-images.githubusercontent.com/64643995/233063839-6215f870-e292-4e86-94e6-67252be74e15.png">